src/styles/isomer-template.scss

```diff
@@ -1,4 +1,6 @@
 @charset "UTF-8";
+@use 'templates/components/hero.scss';
```

Comment:
I realised that changing this file meant that the CSS classes that are only intended for the editor preview gets leaked, which may cause unintended side effects on other aspects of the CMS. Can we put this inside `preview-panel.scss` instead?

Reply:
actually not sure if that's the fix - maybe we should remove the extra import insetad? wdyt

Reply:
@alexanderleegs Do you have more context on why we import `isomer-template.scss` globally in `src/index.jsx`?

Reply:
As discussed offline - we're still using this for preview for regular pages, because that preview doesn't have an isolated set of styles. The compromise we made last time was that all styles imported from `isomer-template` would at leat be prefixed with a unique class, so we don't have leakage

Reply:
Discussed offline, we should not add/modify this file and instead, should just add directly to `preview-panel.scss` if it is a homepage component.

Reply:
:sadge:

Reply:
fixed